nni/retiarii/experiment/pytorch.py

```diff
@@ -354,9 +353,11 @@ def start(self, port: int = 8080, debug: bool = False) -> None:
             log_dir = Path.home() / f'nni-experiments/{self.id}/log'
         nni.runtime.log.start_experiment_log(self.id, log_dir, debug)
 
-        self._proc, self._pipe = launcher.start_experiment_retiarii(self.id, self.config, port, debug)
+        ws_url = f'ws://localhost:{port}/tuner'
+        self._proc = launcher.start_experiment('create', self.id, self.config, port, debug,  # type: ignore
+                                               RunMode.Background, None, ws_url, ['retiarii'])
         assert self._proc is not None
-        assert self._pipe is not None
+        connect_websocket(ws_url)
```

Comment:
When do you disconnect?

Reply:
On exit 😂

Reply:
What if I stop the experiment in the middle the process. When I do something afterwards (e.g., export), would the websocket client still ping-pong?